### DISCUSSION TITLE
Install certain version of Pterodactyl/Wings

### DISCUSSION THREAD

**Discussion by fajrulfx** [vilhelmprytz/pterodactyl-installer#293] [Q&A]
Hi all, how to install a certain version of Pterodactyl/Wings using this installer? I ask this related to the [bug of the latest version of Wings](https://github.com/pterodactyl/panel/issues/3903), so I want to install its previous version. Thank you.

**Comment by Linux123123** [comment]
Install the latest and then just install what matthewpi gave. We only currently support installing the latest version. But maybe in the future #236  will get implemented, which would give a way to install other versions.
